cmd/flowrunner/testdata/flows/all_actions_test.json

```diff
@@ -1,4 +1,5 @@
 {
+  "caller_events": [],
```

Comment:
Looks like while golang does have random map iteration, json encoding of maps is sorted by key.
https://golang.org/src/encoding/json/encode.go